tasks/0-setup-ace-bar.yaml

```diff
@@ -0,0 +1,277 @@
+        REGISTRY_TAG="$(git rev-parse --verify HEAD --short)"
+      fi
+      echo -n "${REGISTRY_SERVER}">$(results.image-server.path)
+      echo -n "${REGISTRY_NAMESPACE}">$(results.image-namespace.path)
+      echo -n "${REGISTRY_REPOSITORY}">$(results.image-repository.path)
+      echo -n "${REGISTRY_TAG}">$(results.image-tag.path)
+      echo -n "${REGISTRY_SERVER}/${REGISTRY_NAMESPACE}/${REGISTRY_REPOSITORY}:${REGISTRY_TAG}">$(results.image-url.path)
+
+
+      if [[ -z "$(params.app-namespace)" ]]; then
+        # Assume current namespace
+        echo -n "${NAMESPACE}">$(results.app-namespace.path)
+      fi
+
+      if [[ -z "$(params.app-name)" ]]; then
+        # Assume image repository/name
+        echo -n "${REGISTRY_REPOSITORY}">$(results.app-name.path)
+      fi
+
+
+      if [[ -n "$(params.image-release)" ]]; then
+        REGISTRY_RELEASE_SERVER_VALUE=$(echo "$(params.image-release)" | awk -F / '{print $1}')
+        if [[ -n "${REGISTRY_RELEASE_SERVER_VALUE}" ]]; then
+          REGISTRY_RELEASE_SERVER="${REGISTRY_RELEASE_SERVER_VALUE}"
+        fi
+
+        REGISTRY_RELEASE_NAMESPACE_VALUE=$(echo "$(params.image-release)" | awk -F / '{print $2}')
+        if [[ -n "${REGISTRY_RELEASE_NAMESPACE_VALUE}" ]]; then
+          if [[ "${REGISTRY_RELEASE_NAMESPACE_VALUE}" = '$NAMESPACE' ]]; then
+            REGISTRY_RELEASE_NAMESPACE=${NAMESPACE}
+          else
+            REGISTRY_RELEASE_NAMESPACE="${REGISTRY_RELEASE_NAMESPACE_VALUE}"
+          fi
+        elif [[ "${REGISTRY_RELEASE_SERVER}" = "image-registry.openshift-image-registry.svc:5000" ]]; then
+          REGISTRY_RELEASE_NAMESPACE=${NAMESPACE}
+        fi
+      fi
+      if [[ -z "${REGISTRY_RELEASE_NAMESPACE}" ]]; then
+        # Assume current k8s namespace
+        REGISTRY_RELEASE_NAMESPACE="${NAMESPACE}"
+      fi
+      if [[ -z "${REGISTRY_RELEASE_SERVER}" ]]; then
+        # Assume internal registry
+        REGISTRY_RELEASE_SERVER="image-registry.openshift-image-registry.svc:5000"
+      fi
+      REGISTRY_RELEASE_URL="${REGISTRY_RELEASE_SERVER}/${REGISTRY_RELEASE_NAMESPACE}/${REGISTRY_REPOSITORY}"
+      echo -n "${REGISTRY_RELEASE_URL}">$(results.image-release.path)
+
+      PERFORM_SCAN="$(params.scan-image)"
+      PERFORM_SCAN_IBM="false"
+      PERFORM_SCAN_TRIVY="false"
+      if [[ "${PERFORM_SCAN}" == "true" ]]; then
+        echo "User selected to perform container image scanning"
+        if echo "$REGISTRY_RELEASE_URL" | grep 'icr.io'; then
+          echo "Using IBM Container registry, enabling scanning using VA"
+          PERFORM_SCAN_IBM="true"
+        else
+          echo "Not using IBM Container registry, enabling scanning using trivy"
+          PERFORM_SCAN_TRIVY="true"
+        fi
+      fi
+
+      echo -n "${PERFORM_SCAN}">$(results.scan-image.path)
+      echo -n "${PERFORM_SCAN_TRIVY}">$(results.scan-trivy.path)
+      echo -n "${PERFORM_SCAN_IBM}">$(results.scan-ibm.path)
+
+      if [[ -d workspace ]]; then
```

Comment:
no need for a new task, add these 4 lines to the normal setup.yaml

Reply:
Consolidated to the standard setup.yaml in latest commit.